tcf_website/management/commands/load_grades.py

```diff
@@ -0,0 +1,398 @@
+import os
+import pandas as pd
+import re
+
+from django.core.management.base import BaseCommand, CommandError
+from tcf_website.models import *
+from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
+
+from tqdm import tqdm
+
+
+class Command(BaseCommand):
+    """
+    How To Use: Run python3 manage.py load_grades to load all grades
+    in the tcf_website/management/commands/grade_data/excel/ directory.
+    This should take ~20 min to run to load all the grades
+    """
+    help = 'Imports grade data from VAGrades Excel Spreadsheets into PostgresSQL database'
+
+    global course_grades
+    global course_instructor_grades
+
+    course_grades = {}
+    course_instructor_grades = {}
+
+    '''
+    def add_arguments(self, parser):
+
+        # Named (optional) arguments
+        parser.add_argument(
+            '--verbose',
+            action='store_true',
+            help='Verbose output',
+        )
+
+        parser.add_argument(
+            'semester',
+            help='Semester to update grades (e.g. "2019_FALL").\nIf you wish to reload all semesters (potentially dangerous!) then put "ALL_DANGEROUS" as the value of this argument.',
+            type=str
+        )
+    '''
```

Comment:
Why is this a docstring? If it's just old code, then it might be good to just remove it?

Reply:
This seems to be code copied and adapted from `load_semester.py`. I think it's there because the functionality is not complete.

Reply:
Done in b11e94fd3d58db1a73892c61ad5720091ceecc7b